app/experimenter/experiments/tests/test_models/test_models_nimbus.py

```diff
@@ -372,6 +372,31 @@ def test_targeting_with_locales_and_countries(self):
             ),
         )
 
+    def test_targeting_with_locales_and_countries_mobile(self):
+        locale_ca = LocaleFactory.create(code="en-CA")
+        locale_ro = LocaleFactory.create(code="ro")
+        country_ca = CountryFactory.create(code="CA")
+        country_ro = CountryFactory.create(code="RO")
+        targeting_config = NimbusExperiment.TargetingConfig
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.LAUNCH_APPROVE_APPROVE,
+            application=NimbusExperiment.Application.FENIX,
+            firefox_min_version=NimbusExperiment.Version.NO_VERSION,
+            firefox_max_version=NimbusExperiment.Version.NO_VERSION,
+            targeting_config_slug=targeting_config.TARGETING_MOBILE_NEW_USER,
+            channel=NimbusExperiment.Channel.NO_CHANNEL,
+            locales=[locale_ca, locale_ro],
+            countries=[country_ca, country_ro],
+        )
+        self.assertEqual(
+            experiment.targeting,
+            (
+                "is_already_enrolled || days_since_install < 7 "
+                "&& locale in ['en-CA'] || ro in locale "
+                "&& locale in ['CA', 'RO']"
```

Comment:
This should read:

```suggestion
                "(is_already_enrolled || days_since_install < 7) "
                "&& (locale in ['en-CA'] || 'ro' in locale) "
                "&& ('CA' in locale || 'RO' in locale)"
```

NB: I also noticed `ro` should  be quoted.

Reply:
so it must have the parentheses?

Reply:
:+1: yes.

Reply:
Oh I see, this is an entirely separate issue.  We should file this separately.

Reply:
Filed https://github.com/mozilla/experimenter/issues/7120